YvesIraguha/readit_react_front_end
# #165466436 Sign Up

#### What does this PR do?
- Write tests for all components
#### Description of Task to be completed?
- Write tests for all components
- Refactor loading environment variables, both in tests and in development 
#### How should this be manually tested?
- Clone the repo and check out the branch `ft-sign-up-165466436`
- Run all tests, they should pass
- Start servers, back-end, and front-end. You should see articles being displayed on the home page, and you should be directed to one article when you click on one of the articles being displayed. 

#### What are the relevant pivotal tracker stories?
#165466436
